subroutines.sh

```diff
@@ -89,9 +87,14 @@ check_presence() {
         log_error "$MQTT_OUT" &&
         return
       log_debug "mqtt topic $MQTT_TOPIC succesfully updated to OFF"
+      # Update presence expire time
```

Comment:
I'm not sure it's a good idea to increase the expiration time when the presence is not detected. If we do that, when the car gets home, this increases the time to toggle the state to Home by as much as 2m10s. With the new ble presence, once written, the increase will go as high as 4m10s.

ps: the 10s is the current bluetoothctl presence detection timeout.

Reply:
Ok thanks I'll drop it from the non detected case.

Reply:
OK I removed from the non-detected case